src/qililab/instruments/qblox/qblox_qrm.py

```diff
@@ -254,7 +254,16 @@ def get_acquisitions(self) -> QbloxResult:
                 if sequencer.scope_store_enabled:
                     self.device.store_scope_acquisition(sequencer=sequencer_id, name="default")
 
-                results.append(self.device.get_acquisitions(sequencer=sequencer.identifier)["default"]["acquisition"])
+                for key, data in self.device.get_acquisitions(sequencer=sequencer.identifier).items():
+                    acquisitions = data["acquisition"]
+                    # parse acquisition index
+                    qubit, measure = key.split("_")[1:]
```

Comment:
If you want to discard the first element you can also do `_, qubit, measure = key.split("_")`. Personally I find it more readable than the magic index.